packages/io/stl-deserializer/README.md

```diff
@@ -1,13 +1,13 @@
 ## @jscad/stl-deserializer
 
-> stl deserializer for the jscad project
+> Deserializer of STL data to CSG geometries
```

Comment:
not csg anymore ? so perhaps '3d geometry for the jscad project' ? something else ?